ecoscope_workflows/tasks/results/_map_config.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel
+
+
+class LayerStyleProperty(BaseModel):
+    auto_highlight: bool = False
+    opacity: float = 1
+    pickable: bool = True
+
+
+class PathLayerProperty(LayerStyleProperty):
+    get_color: list[int] = [0, 0, 0, 255]
```

Comment:
This allows only a single color per map layer. We definitely want to support having per-row colors as well, something like:

```suggestion
    get_color: list[int] | list[list[int]] = [0, 0, 0, 255]
```